### PR TITLE
Renaming "Opt-in" to "Requires Confirmation"

### PR DESCRIPTION
**Pull request by PeerRich**
quite a lot of customers are not sure what "opt-in" means, especially in non-english translations

### CODE REVIEW

File: apps/web/public/static/locales/ar/common.json [hunk 1]
```diff
@@ -615,7 +615,7 @@
   "disable_notes": "إخفاء الملاحظات في التقويم",
   "disable_notes_description": "لأسباب تتعلق بالخصوصية، سيتم إخفاء المدخلات والملاحظات الإضافية في إدخال التقويم. سيتم إرسالها إلى بريدك الإلكتروني.",
   "opt_in_booking": "حجز يتطلب موافقة",
```
Comment (baileypumfleet) [inline apps/web/public/static/locales/ar/common.json:617]: You left the opt in booking string in, and only changed the name of the description string

File: apps/web/public/static/locales/cs/common.json [hunk 2]
```diff
@@ -614,7 +614,7 @@
   "disable_notes": "Skrýt poznámky v kalendáři",
   "disable_notes_description": "Z důvodu ochrany soukromí budou v záznamu kalendáře skryty další vstupy a poznámky. Stále budou odeslány na váš e-mail.",
   "opt_in_booking": "Přidat se k rezervaci",
```
Comment (baileypumfleet) [inline apps/web/public/static/locales/cs/common.json:616]: Again

File: apps/web/public/static/locales/de/common.json [hunk 3]
```diff
@@ -615,7 +615,7 @@
   "disable_notes": "Notizen im Kalender ausblenden",
   "disable_notes_description": "Aus Datenschutzgründen werden zusätzliche Eingaben und Notizen im Kalendereintrag ausgeblendet. Sie werden trotzdem an Ihre E-Mail gesendet.",
```
Comment (baileypumfleet) [inline apps/web/public/static/locales/de/common.json:616]: Again

File: apps/web/public/static/locales/pl/common.json [hunk 4]
```diff
@@ -613,8 +613,7 @@
   "add_input": "Dodaj Wejście",
   "disable_notes": "Ukryj notatki w kalendarzu",
   "disable_notes_description": "Ze względu na prywatność, dodatkowe dane wejściowe i notatki zostaną ukryte we wpisie kalendarza. Będą one nadal wysyłane na Twój adres e-mail.",
-  "opt_in_booking": "Opt-in Rezerwacji",
```
Comment (baileypumfleet) [inline apps/web/public/static/locales/pl/common.json:616]: Again

**Review by pumfleet — CHANGES_REQUESTED** [review]
There are missing strings throughout the whole PR - see comments for more

**Pull request closed without merging**